pkg/image/api/dockertypes.go

```diff
@@ -68,8 +83,8 @@ type DockerImageManifest struct {
 	History      []DockerHistory `json:"history"`
 
 	// schema2
-	Layers []distribution.Descriptor `json:"layers"`
```

Comment:
Seems like we'd want a reflective drift check like we did for openshift/kube authorization types.